.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9-dev, pypy3]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
I've never ran the tests in other OSes in CI. I knew it worked in macOS because this is what I use to develop, and the CI runs in a linux container in travis. Thank you for including a multi-OS build here. What about step up our game here and see what happens in `windows-latest`?

Reply:
Yes, it's great to be able to include multiple operating systems on GHA!

I had tried `windows-latest`, but it's failing:

```
2020-08-02T08:41:20.9213149Z ================================== FAILURES ===================================
2020-08-02T08:41:20.9213407Z _________ TestReport.test_should_override_test_titles_with_title_mark _________
2020-08-02T08:41:20.9213596Z 
2020-08-02T08:41:20.9213801Z self = <tests.test_plugin.TestReport object at 0x000002F2B0A0E160>
2020-08-02T08:41:20.9214083Z testdir = <Testdir local('C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-runneradmin\\pytest-0\\test_should_override_test_titles_with_title_mark0')>
2020-08-02T08:41:20.9214299Z 
2020-08-02T08:41:20.9215109Z     def test_should_override_test_titles_with_title_mark(
2020-08-02T08:41:20.9215341Z         self,
2020-08-02T08:41:20.9215556Z         testdir
2020-08-02T08:41:20.9215770Z     ):
2020-08-02T08:41:20.9216096Z         testdir.makefile('.py', test_module_name="""
2020-08-02T08:41:20.9216275Z             import pytest
2020-08-02T08:41:20.9216466Z     
2020-08-02T08:41:20.9216663Z             @pytest.mark.{}('''
2020-08-02T08:41:20.9216860Z                 My Title
2020-08-02T08:41:20.9217229Z                 My precious title
2020-08-02T08:41:20.9217473Z             ''')
2020-08-02T08:41:20.9217678Z             def test_a_passing_test():
2020-08-02T08:41:20.9217875Z                 assert True
2020-08-02T08:41:20.9218064Z         """.format(
2020-08-02T08:41:20.9218257Z             constants.TITLE_MARK
2020-08-02T08:41:20.9218451Z         ))
2020-08-02T08:41:20.9218614Z     
2020-08-02T08:41:20.9218811Z         result = testdir.runpytest('--force-testdox')
2020-08-02T08:41:20.9219186Z     
2020-08-02T08:41:20.9219412Z >       assert 'My Title\n   My precious title' in result.stdout.str()
2020-08-02T08:41:20.9219769Z E       AssertionError: assert 'My Title\n   My precious title' in '============================= test session starts =============================\nplatform win32 -- Python 3.8.5, pyte...                             [100%]\n\n============================== 1 passed in 0.03s =============================='
2020-08-02T08:41:20.9220113Z E        +  where '============================= test session starts =============================\nplatform win32 -- Python 3.8.5, pyte...                             [100%]\n\n============================== 1 passed in 0.03s ==============================' = <bound method LineMatcher.str of <_pytest.pytester.LineMatcher object at 0x000002F2B09F6FD0>>()
2020-08-02T08:41:20.9220451Z E        +    where <bound method LineMatcher.str of <_pytest.pytester.LineMatcher object at 0x000002F2B09F6FD0>> = <_pytest.pytester.LineMatcher object at 0x000002F2B09F6FD0>.str
2020-08-02T08:41:20.9221996Z E        +      where <_pytest.pytester.LineMatcher object at 0x000002F2B09F6FD0> = <RunResult ret=ExitCode.OK len(stdout.lines)=13 len(stderr.lines)=0 duration=0.11s>.stdout
2020-08-02T08:41:20.9222447Z 
2020-08-02T08:41:20.9222932Z d:\a\pytest-testdox\pytest-testdox\tests\test_plugin.py:156: AssertionError
```

https://github.com/hugovk/pytest-testdox/runs/937149823?check_suite_focus=true